### DISCUSSION TITLE
Is 'per-client' transcoding supported?

### DISCUSSION THREAD

**Discussion by Libarch** [gerbera/gerbera#1790] [Q&A] [answered]
Hello, I have two media renderers / clients. One of them can play FLAC files, the other (an old Samsung Series D TV) cannot. Is it possible to enable transcoding for a specific client, but not generally?

**Comment by KarlStraussberger** [comment, marked as answer]
It's actually one of the new features in 1.9.

Check out `client-flags` under https://docs.gerbera.io/en/stable/config-transcode.html#profiles
